Makefile.include

```diff
@@ -1,6 +1,8 @@
 .PHONY: default init release gen test clean all
 default: help
 
+PMM_RELEASE_PATH ?= ./bin
```

Comment:
Why do you prefer variable for that? Do we need to overwrite it somewhere?

Reply:
Yes, as I mentioned above we are not able to use even tools outside of dev container on systems with arm architecture.